src/syncgitlab2msproject/gitlab_issues.py

```diff
@@ -266,9 +278,9 @@ def get_group_id_from_gitlab_project(project: Project) -> Optional[int]:
 
 def get_gitlab_class(server: str, personal_token: Optional[str] = None) -> Gitlab:
     if personal_token is None:
-        return Gitlab(server)
+        return Gitlab(server, ssl_verify=False)
     else:
-        return Gitlab(server, private_token=personal_token)
+        return Gitlab(server, private_token=personal_token, ssl_verify=False)
 
```

Comment:
As written in before: Don't do this by default. You can add an option that disables this changes so.